src/ring_theory/algebra_operations.lean

```diff
@@ -279,6 +279,24 @@ lemma mem_div_iff_smul_subset {x : A} {I J : submodule R A} : x ∈ I / J ↔ x
   λ h y hy, h (set.smul_mem_smul_set hy) ⟩
 
 lemma le_div_iff {I J K : submodule R A} : I ≤ J / K ↔ ∀ (x ∈ I) (z ∈ K), x * z ∈ J := iff.refl _
+
+@[simp] lemma map_div {B : Type*} [comm_ring B] [algebra R B]
+  (I J : submodule R A) (h : A ≃ₐ[R] B) :
+  (I / J).map h.to_linear_map = I.map h.to_linear_map / J.map h.to_linear_map :=
```

Comment:
What prompted the coercion comment above was this line. It would be cool if you could just get away with `h`.

Reply:
Perhaps current mathlib convention is no coercion.